au3/libraries/lib-track/Track.h

```diff
@@ -78,8 +78,8 @@ template<typename T>
 class TrackId
 {
 public:
-   TrackId() : mValue(-1) {}
-   explicit TrackId (long value) : mValue(value) {}
+    TrackId() = default;
+   explicit TrackId (int64_t value) : mValue(value) {}
 
    bool operator == (const TrackId &other) const
```

Comment:
Add the missing comparison operators. Defining only == and < leaves TrackId with incomplete comparisons.